server/db.js

```diff
@@ -82,6 +82,20 @@ export function create(config) {
           PRIMARY KEY (id),
           UNIQUE(username)
       )`,
+    `CREATE TABLE IF NOT EXISTS "pets" (
+            id serial NOT NULL,
+            name varchar(40) NOT NULL,
+            owner integer references users(id),
+            birthdate date NOT NULL,
+            breed varchar(40) NOT NULL,
+            PRIMARY KEY (id)
+        )`,
+    `CREATE TABLE IF NOT EXISTS "likes" (
+            pet1 integer references pets(id),
```

Comment:
Pone algo como `from` y `to` así queda más explicito quien le dio *like* a quien